eclass/ecm.eclass

```diff
@@ -664,6 +667,33 @@ ecm_src_install() {
 			fi
 		done
 	fi
+
+	mv_metainfo() {
+		if [[ -f ${1} ]]; then
+			mv -v ${1} ${1/${2}/${3}} || die
+		fi
+	}
+
+	if [[ -n ${_KDE_ORG_ECLASS} && -d "${ED}"/usr/share/metainfo/ ]]; then
+		if [[ ${KDE_ORG_NAME} != ${PN} ]]; then
+			local ecm_metainfo
+			pushd "${ED}"/usr/share/metainfo/ > /dev/null || die
+			for ecm_metainfo in ls -1 *metainfo.xml; do
```

Comment:
This is too prone to issues with locale and such, please use `find` with `-iname`?